tests/integrate/102_PW_DS_davsubspace_sca/INPUT

```diff
@@ -0,0 +1,29 @@
+INPUT_PARAMETERS
+#Parameters (1.General)
+suffix			autotest
+calculation     scf
+
+nbands			6
+symmetry		1
+pseudo_dir	../../PP_ORB
+
+#Parameters (2.Iteration)
+ecutwfc			20
+scf_thr				1e-8
+scf_nmax			100
+
+
+#Parameters (3.Basis)
+basis_type		pw
+
+#Parameters (4.Smearing)
+smearing_method		gauss
+smearing_sigma			0.002
+
+#Parameters (5.Mixing)
+mixing_type		plain
+mixing_beta		0.5
+
+ks_solver       dav_subspace
+diag_subspace 2
+nb2d 4
```

Comment:
![image](https://github.com/user-attachments/assets/e5fc69b0-6585-4e8b-9acf-3dbfd537680d)

In my understanding, this parameter is related to `elpa` or `scalapack`, not strongly bound to lcao. Do we still need to adjust the input document.